### PR TITLE
 fix hash_with_views error

### PR DESCRIPTION
**Pull request by bgoldberg-habana**
fix hash_with_views error
this api is no longer valid at next release

### CODE REVIEW

**Comment by regisss** [comment]
"next release" means SynapseAI v1.14?

**Comment by bgoldberg-habana** [comment]
> "next release" means SynapseAI v1.14?

yes, we remove this parameter from api.

**Comment by regisss** [comment]
> > "next release" means SynapseAI v1.14?
> 
> yes, we remove this parameter from api.

Okay, so we should wait for 1.14 to be released before merging this PR right?

**Comment by bgoldberg-habana** [comment]
> > > "next release" means SynapseAI v1.14?
> > 
> > 
> > yes, we remove this parameter from api.
> 
> Okay, so we should wait for 1.14 to be released before merging this PR right?

no, we need it now for internal use..

File: examples/text-generation/utils.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def setup_model(args, model_dtype, model_kwargs, logger):
     if args.use_hpu_graphs:
         from habana_frameworks.torch.hpu import wrap_in_hpu_graph
 
-        if check_habana_frameworks_min_version("1.13.0"):
+        if check_habana_frameworks_version("1.13.0"):
```
Comment (regisss) [inline examples/text-generation/utils.py:135]:
```suggestion
        # TODO: remove the following check from SynapseAI v1.15
        if check_habana_frameworks_version("1.13.0"):
```

**Review by regisss — COMMENTED** [review]
Okay I see. Let's just add a comment to remove this check at some point (when 1.15 is released I think).

**Review by libinta — APPROVED** [review]

**Review by regisss — APPROVED** [review]
LGTM

**Pull request merged**